src/main/fc/rc.c

```diff
@@ -132,6 +133,11 @@ float getRcDeflectionAbs(int axis)
     return rcDeflectionAbs[axis];
 }
 
+float getMaxRcDeflectionAbs(void)
+{
```

Comment:
```suggestion
float getMaxRcDeflectionAbs(void) {
```

Please see: https://betaflight.com/docs/development/CodingStyle#opening-braces

Reply:
Functions should have the opening brace on the following line according to the coding style docs, so the current version looks correct to me.